### PR TITLE
WaaS

### PR DESCRIPTION
**Pull request by pkieltyka**

### CODE REVIEW

File: relayer/rpc_relayer.go [hunk 1]
```diff
@@ -127,12 +129,44 @@ func (r *RpcRelayer) Relay(ctx context.Context, signedTxs *sequence.SignedTransa
...
 	}
 
-	// send to guest module if factory address is the recipient (in order to deploy wallet)
-	// todo: split wallet create and other transactions
-	for _, txn := range signedTxs.Transactions {
-		if txn.To == signedTxs.WalletContext.FactoryAddress {
+	if r.IsDeployTransaction(signedTxs) {
+		to = signedTxs.WalletContext.GuestModuleAddress
+	} else {
+		isDeployed, err := sequence.IsWalletDeployed(r.GetProvider(), to)
```
Comment (pkieltyka) [inline relayer/rpc_relayer.go:135]: we should add an in-memory cache here.. as testing this each time, is going to be slow and resource intensive

Comment (marino39) [reply relayer/rpc_relayer.go]: This is now done in api waas. So I have removed it from here.

File: signing_service/signing_service.go [hunk 2]
```diff
@@ -7,7 +7,6 @@ import (
```
Comment (pkieltyka) [inline signing_service/signing_service.go:7]: are we still using the "signing_service" for anything, or can we remove this code...?

Comment (marino39) [reply signing_service/signing_service.go:7]: I believe we can safely delete it now.

**Pull request merged**